### PR TITLE
Use the new ServiceWidget

### PR DESCRIPTION
**Pull request by dgdavid**
More info: https://trello.com/c/uAe4i9Ru/107-5-fate319428-allow-socket-activation-widget

| From | To |
|--------|----|
| iscsi-lio-server_old | iscsi_new |

### CODE REVIEW

**Comment by ls-zhu** [comment]
@dgdavid Thanks for your commit, would you please add a changelog entry?

**Comment by dgdavid** [comment]
@ls-zhu this is a work in progress PR. I mean, a code review is need and probably changes are not be available for the maintenance update.

I will to update it as soon as I have a better idea about the right version (I think it will be `4.1.0`). If you want, I will to ping back to you when it is finished. Meanwhile, please consider it as pending.

Thank you :)

**Comment by dgdavid** [comment]
@ls-zhu, @joseivanlopez 

Version and changelog updated

File: src/include/iscsi-lio-server/UI_dialogs.rb [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
+require 'yast2/execute'
+require 'yast2/system_service'
+
 require 'cwm/table'
 require 'cwm/dialog'
-require 'yast2/execute'
+require 'cwm/widget'
+require 'cwm/service_widget'
+
+require_relative './iscsi-lio-server_helper.rb'
```
Comment (imobachgs) [inline src/include/iscsi-lio-server/UI_dialogs.rb]: Why `require_relative` is used here?

File: src/include/iscsi-lio-server/UI_dialogs.rb [hunk 2]
```diff
@@ -1,23 +1,21 @@
...
+require_relative './TargetData.rb'
```
Comment (imobachgs) [inline src/include/iscsi-lio-server/UI_dialogs.rb]: Why `require_relative`? Please, unless it is not possible for some weird reason, use `require`.

**Pull request merged**